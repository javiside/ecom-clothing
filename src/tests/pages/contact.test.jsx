import React from "react";
import { shallow } from "enzyme";

import Contact from "../../pages/contact";

describe("Contact", () => {
  it("should mount the component correctly", () => {
    expect(shallow(<Contact />)).toMatchSnapshot();
  });
});
