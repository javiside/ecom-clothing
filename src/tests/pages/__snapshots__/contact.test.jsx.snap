// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Contact should mount the component correctly 1`] = `
<div
  className="LI-badge-container-vertical-light LI-badge-container vertical light large"
  dir="ltr"
>
  <div
    className="LI-profile-badge-header LI-name-container"
  >
    <div
      className="LI-col"
    >
      <img
        alt="background"
        className="LI-profile-pic-container"
        src="bg.jpg"
      />
      <img
        alt="Francisco M."
        className="LI-profile-pic"
        src="pic.jpg"
      />
    </div>
    <div
      className="LI-col LI-header"
    >
      <div
        className="LI-name"
      >
        <a
          href="https://mx.linkedin.com/in/javiside?trk=profile-badge-name"
        >
          Francisco M.
        </a>
      </div>
      <div
        className="LI-title"
      >
        Software Engineer
      </div>
    </div>
  </div>
  <ul
    className="more-info"
  >
    <li
      className="LI-field"
    >
      <a
        href="https://www.linkedin.com/company/walmartecommerce?trk=profile-badge-company"
      >
        Walmart eCommerce
      </a>
    </li>
    <li
      className="LI-field"
    >
      Sonora State University
    </li>
  </ul>
  <div
    className="LI-profile-badge-footer"
  >
    <a
      className="LI-view-profile"
      href="https://mx.linkedin.com/in/javiside?trk=profile-badge-cta"
    >
      View profile
    </a>
    <span
      className="LI-logo"
    >
      <img
        alt="LinkedIn"
        className="LI-icon"
        src="li.png"
      />
    </span>
  </div>
</div>
`;
